packages/components/stamp/src/stamp.tsx

```diff
@@ -133,6 +148,17 @@ const getStampStyles = (props: StylesFunctionParams) => {
 const Stamp = (props: Props) => {
   const { themedValue } = useTheme();
   const overrideTextColor = themedValue(false, true);
+  const Icon =
+    props.icon &&
+    cloneElement(props.icon, {
+      size: 'medium',
+      color: getIconColor(props, overrideTextColor),
+    });
+
+  warning(
+    !props.children,
+    'Stamp: The `children` prop has been deprecated. Please use the `label` and `icon` prop to render the content.'
+  );
```

Comment:
I think we can use both since this one just warns the user that old implementation is deprecated.
```js
  warning(
    !props.children,
    'Stamp: The `children` prop has been deprecated. Please use the `label` and `icon` prop to render the content.'
  );
```
while this one seems to check if the component is used correctly (provide either one of the props expected) 
```js
   warning(
    hasContent,
    [
      'ui-kit/Stamp:',
      '"label" is required when "children" is `undefined` and vice versa.',
      'Provide "label" or "children"',
    ].join(' ')
  );
```
What do you think ?

Reply:
Ah nevermind, we only want to deprecate the `children` prop yes. All good then, thanks for double checking!